ckanext/google_tag_manager/helpers.py

```diff
@@ -0,0 +1,8 @@
+from ckan.common import config
+from builtins import str
```

Comment:
Is this still correct after what we talked about this week? Maybe this should be `six.text_type`?

Reply:
Good catch, I haven't revisited this since last week. I'll update to try to make it compatible with both.